crypto/ledger/ledger_mock.go

```diff
@@ -10,7 +10,6 @@ import (
 
 	secp256k1 "github.com/tendermint/btcd/btcec"
 	"github.com/tendermint/tendermint/crypto"
-	tmsecp256k1 "github.com/tendermint/tendermint/crypto/secp256k1"
```

Comment:
@aaronc what's the best to way to get mocks use sdk packages

Reply:
Look at `mockgen` in the Makefile